### PR TITLE
MRG: Fix plot_montage

### PR DESCRIPTION
**Pull request by larsoner**
Eventually will fix what #5675 works around. This fixes that failure but introduces another. I see how to fix it but won't have time for a few hours.

- [x] fix master bug
- [x] fix introduced bugs
- [x] add check and error for all `np.nan` positions, plus test for all `np.nan`

So @massich it's probably not worth merging #5675

### CODE REVIEW

**Comment by larsoner** [comment]
This is ready for review/merge from my end. Tests pass locally so I'm optimistic about CIs

**Comment by massich** [comment]
Ok, (it works on my machine) hopefully this should make travis green.

**Comment by massich** [comment]
Appveyor will be red, but if [This travis](https://travis-ci.org/mne-tools/mne-python/builds/447888641?utm_source=github_status&utm_medium=notification) is green we should get it in.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -110,6 +110,8 @@ Bug
 
 - Fix bug with channel names in ``mgh70`` montage in :func:`mne.channels.read_montage` by `Eric Larson`_
 
+- Fix check in :func:`mne.vis.plot_sensors` for inalid channel locations by `Eric Larson`_
```
Comment (agramfort) [inline doc/whats_new.rst]: inalid -> invalid

**Comment by agramfort** [comment]
thx @larsoner

**Pull request merged**